python/setup.py

```diff
@@ -1,9 +1,33 @@
 #!/usr/bin/python
+import io
+import os
+
 from setuptools import setup
 
+
+def read_from(file):
+    reply = []
+    with io.open(os.path.join(here, file), encoding='utf8') as f:
+        for l in f:
+            l = l.strip()
+            if not l:
+                break
+            if l[:2] == '-r':
+                reply.extend(read_from(l.split(' ')[1]))
+                continue
+            if l[0] != '#' or l[:2] != '//':
+                reply.append(l)
+    return reply
+
+
+here = os.path.abspath(os.path.dirname(__file__))
+with io.open(os.path.join(here, 'README.rst'), encoding='utf8') as f:
+        README = f.read()
+
+
 setup(
     name='http_ece',
-    version='0.5.0',
+    version='0.6.0',
     author='Martin Thomson',
```

Comment:
Thanks. PyPi can get a bit squirrelly if you have multiple authors or emails, so I'll just stick with the participants list on github. Plus, this way you look even better, so bonus on that front.